### ISSUE TITLE
document usage of yargs with input tools like inquirer

### ISSUE THREAD

**Issue by bcoe** [yargs/yargs#1476]
It would be good to add a section to our docs that demonstrates using yargs in parallel with tools like [inquirer](https://www.npmjs.com/package/inquirer).

see: https://github.com/yargs/yargs/issues/286

**Comment by akashroy1** [comment]
Can I work on this issue?

**Comment by bcoe** [comment]
@akashroy1 I would love help with this. I would add it as an [Advanced topic](https://github.com/yargs/yargs/blob/main/docs/advanced.md).

**Comment by shadowspawn** [comment]
Yes, I think so. Thanks @nephh. Benjamin is busy elsewhere, but I can give you feedback and review.

**Comment by nephh** [comment]
Thanks @shadowspawn, I just went ahead and made a PR: #2385 This is my first time contributing to an open source project, so please let me know if there's anything I can do differently or any changes that should be made!

**Comment by shadowspawn** [comment]
#2385 has been merged.

Thank you for your contributions.
